src/QuadForm/Torsion.jl

```diff
@@ -2159,3 +2160,79 @@ underlying abelian group is in Smith normal form.
 """
 is_snf(T::TorQuadModule) = is_snf(abelian_group(T))
 
+################################################################################
+#
+#  Isotropic check
+#
+################################################################################
+
+@doc raw"""
+    is_totally_isotorpic(T::TorQuadModule)
+
+Return whether the quadratic form on the torsion quadratic module `T` vanishes.
+"""
+function is_totally_isotropic(T::TorQuadModule)
+  for a in gens(T)
+    if !is_zero(quadratic_product(a))
+      return false
+    end
+    for b in gens(T)
+      if !is_zero(a * b)
```

Comment:
```suggestion
      k = inner_product(ambient_space(cover(T)), lift(a), lift(b))
      if !is_zero(value_module_quadratic_form(T)(2*k))
```

Reply:
I would guess that in the general case, it may happen that `a` and `b` are not orthogonal but `2*a*b` is zero for the quadratic modulus (for equal moduli for instance)